code-challenges.test.js

```diff
@@ -27,6 +59,20 @@
 
 // --------------------2) Create a function that takes in an array and returns a new array of only odd numbers sorted from least to greatest.
 
+describe("onlyOdds", () => {
+
+    it("returns a new array of only odd numbers sorted from least to greatest", () => {
+    
+    const fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
+  
+    const fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
+
+      expect(onlyOdds(fullArr1)).toEqual([-9, 7, 9, 199])
+      expect(onlyOdds(fullArr2)).toEqual([-823, 7, 23])
+    })
+  })
+
+
 // a) Create a test with expect statements for each of the variables provided.
 
 const fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
```

Comment:
Because you have already placed the variables in your test, you don't need to declare them again. The test suite will keep track of them.